### DISCUSSION TITLE
Games shortcuts

### DISCUSSION THREAD

**Discussion by VellarIFSUL** [rafaelvcaetano/melonDS-android#967] [Ideas]
Would be a very nice addition if we can have the ability to put out game shortcuts to our home screen, even better if the already shown game icon inside the emulator could be the shortcut icon

**Comment by rafaelvcaetano** [comment]
This is already possible. They are added through your home screen. Go to the screen where you choose widgets and you will find one called "ROM shortcut" for melonDS. Add it to your screen and you will be prompted to select the game that you want the shortcut to point to.

**Reply by VellarIFSUL** [reply]
My mistake, sorry! Thanks to showme how to do
